package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "konflux-ci-docs",
+  "name": "konflux-upstream-docs",
```

Comment:
Was this change intentional?

Reply:
guess it was due to my repo name...fixed